downstream/modules/platform/con-install-scenario-examples.adoc

```diff
@@ -9,8 +9,10 @@ Red Hat supports a number of installation scenarios for {PlatformNameShort}. Rev
 [IMPORTANT]
 ====
 * For Red Hat Ansible Automation Platform or automation hub: Add an automation hub host in the [automationhub] group.
-* You cannot install {ControllerName} and {HubName} on the same node.
-* Provide a reachable IP address or fully qualified domain name (FDQN) for the [automationhub] host to ensure users can sync and install content from automation hub from a different node. Do not use 'localhost'.
+* You cannot install {ControllerName} and {HubName} on the same node for versions of {PlatformNameShort} before version {PlatformVers}.
```

Comment:
For AAP 2.3+:  Although you can install both controller and hub on the same node, this is designed for non-production use cases only. It is highly discouraged to use this type of install in a production or customer environment due to the potential for heavy resources and contention

Can you add something along this line?